web_scraper/load.py

```diff
@@ -161,16 +199,26 @@ def save_html_css(current_url: str) -> str:
                        aws_secret_access_key=environ["AWS_SECRET_ACCESS_KEY"])
     print(f"Connected to S3 --- {perf_counter() - startup}s.")
 
-    list_of_urls = ["https://eveninguniverse.com/fiction/the-meteor-generation.html", "https://www.youtube.co.uk", "https://www.youtube.co.uk/"]
+    list_of_urls = ["https://eveninguniverse.com/fiction/the-meteor-generation.html",
+                    "https://www.youtube.co.uk", "https://www.youtube.co.uk/"]
 
     download = perf_counter()
-    print("Uploading HTML and CSS data to S3...")
+    print("Uploading HTML and image data to S3...")
     for url in list_of_urls:
 
-        html_file_name, css_file_name = save_html_css(url)
-        upload_to_s3(client, url,
-                     html_file_name, css_file_name)
+        soup = get_soup(url)
+        title = extract_title(url)
+        domain = extract_domain(url)
+        timestamp = datetime.utcnow().isoformat()
+        html_file_name = process_html_content(soup, domain, title, timestamp, client)
+        img_file_name = process_screenshot(url, domain, title, timestamp, client, hti)
+        css_file_name = process_css_content(soup, domain, title, timestamp, client)
+
+        response_data = {"scrape_at": timestamp, "html_s3_ref": html_file_name,
+                        "css_s3_ref": css_file_name, "screenshot_s3_ref": img_file_name,
+                        "is_human": IS_HUMAN}
 
-    shutil.rmtree("static/")
+        if html_file_name and img_file_name and css_file_name:
```

Comment:
Nice final if statement to check only successfully scraped websites will be uploaded